microsoft-edge/extensions-chromium/whats-new/roadmap.md

```diff
@@ -45,21 +45,6 @@ See [Develop an extension for the Microsoft Edge sidebar](../developer-guide/sid
 
 Users of your extension will be able to report issues through the extension's webpage on the Microsoft Edge Add-ons website.  Developer support information will be shown on the product page of an extension.  This information will allow users to directly reach out to the extension developer for extension-specific support.
 
```

Comment:
todo: construct the reviewer guide in PR main page, examine context.  any other articles to make equivalent fix?